source/welcome/whats-new-in-v72.rst

```diff
@@ -136,6 +140,11 @@ Boards that you create after the migration won’t be linked to a workspace, and
 Frequently asked questions
 --------------------------
 
+Do I need to plan for downtime during the migration?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are schema changes in v7.2. If you have a large database you may need to allocate additional time and resources to this migration. Please review `this document <https://docs.google.com/document/d/1hKQV6QfXSJ6LznvKxoENvU1Z21ulaZ09b-h8N1caS64/edit#heading=h.1ozc9hcotgot>`_ for details around permission migrations.
```

Comment:
Is the content duplication here intentional?

Reply:
Yep. The original GitHub issue indicated that this should be in the doc and the FAQ.